### DISCUSSION TITLE
[BUG] `files viewed` indicator error during code review

### DISCUSSION THREAD

**Discussion by Jef-Z** [community/community#65076] [Pull Requests]
### Select Topic Area

Bug

### Body

I am doing a code review. I will click the `Viewed` button on the right side of the file. But after I clicked on it sometime, my internet connection got disconnected. Then I saw the indicator error displayed:
![image](https://github.com/community/community/assets/45141759/24403ade-39ff-4747-9ca6-6c7bb1e514d0)

It seems that the timing of updating the indicator is not right. 
Because when I clicked the `Viewed` button, the `files viewed` indicator was updated immediately. But the `Viewed` button is still greyed out. Maybe this means that the request hasn't been successful yet.

**Comment by Shednandezstore** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> I am doing a code review. I will click the `Viewed` button on the right side of the file. But after I clicked on it sometime, my internet connection got disconnected. Then I saw the indicator error displayed:
> 
> ![image](https://github.com/community/community/assets/45141759/24403ade-39ff-4747-9ca6-6c7bb1e514d0)
> 
> 
> 
> It seems that the timing of updating the indicator is not right. 
> 
> Because when I clicked the `Viewed` button, the `files viewed` indicator was updated immediately. But the `Viewed` button is still greyed out. Maybe this means that the request hasn't been successful yet.

[**@shednandezstore**]()
